yt/frontends/boxlib/data_structures.py

```diff
@@ -1405,6 +1403,16 @@ def _read_header(raw_file, field):
                 lo_corner, hi_corner, node_type = _line_to_numpy_arrays(clean_line)
                 boxes.append((lo_corner, hi_corner, node_type))
 
+            try:
+                # nghost_line is a single number
+                ng = int(nghost_line[0])
+                ndims = len(lo_corner)
+                nghost = np.array(ndims * [ng])
+            except ValueError:
+                # nghost_line is (#,#,#)
+                nghost_list = nghost_line[0][1:-1].split(",")
+                nghost = np.array([int(ng) for ng in nghost_list])
```

Comment:
while we're at it, might as well make this more idiomatic to numpy
```suggestion
                nghost = np.array(nghost_list, dtype="int64")
```